setup.py

```diff
@@ -10,7 +10,7 @@ def text_from_file(path):
     "coverage",
     "isort",
     "jsonschema",
-    "pytest<8.0",  # pytest >= 8 is incompatible with pytest-black
+    "pytest<9.0",  # pytest >= 8 is incompatible with pytest-black
```

Comment:
```suggestion
    "pytest",
```

Reply:
Based on CI passing, it appears that the issue with `pytest >= 8` and `pytest-black` is resolved. No need to keep the restriction or comment.